src/pages/index.tsx

```diff
@@ -1,16 +1,16 @@
-import { PageHead } from "@/common/components/PageHead";
+import { IconBox } from '@/common/components/IconBox';
+import { PageHead } from '@/common/components/PageHead';
+import { Box } from '@chakra-ui/react';
 
 export default function Home() {
   return (
-    <div className="h-full">
+    <Box minH="100%">
```

Comment:
This might create conflict in my recent PR. :D

Reply:
oh yeah, many conflicts actually :P

Reply:
I can help you resolve. just lmk ;)

Reply:
no problem, i can do it!